sdk/src/main/java/io/radar/sdk/RadarLocationReceiver.kt

```diff
@@ -98,6 +98,9 @@ class RadarLocationReceiver : BroadcastReceiver() {
 
     @SuppressLint("MissingPermission")
     override fun onReceive(context: Context, intent: Intent) {
+        if (!Radar.initialized) {
```

Comment:
@phil-brown Can we do 2 things:
1) write a unit test that fails without this fix
2) add a comment:
```
// Radar must be initialized before using Radar.logger
// TODO better factoring of RadarLogger
```

Reply:
Test added. Comment left out, as per offline discussion.